python/cufinufft/tests/test_basic.py

```diff
@@ -7,21 +9,30 @@
 
 import utils
 
+# NOTE: Tests below fail for shapes of size (16, 16) and tolerances 1e-4.
```

Comment:
That's weird. I'd expect more accuracy for smaller N1,N2,N3.   But I checked CPU code and requesting 1e-4 gets as bad as 7e-3 for single-prec (16,16) w/ M=1024. In short, 10.tol is too stringent in general, see below.

Reply:
Got it. Relaxed to `25 * tol` and it seems to work fine now (strangely enough, however, the “simple” interface seems to be fine with `10 * tol`).

Reply:
(So there was a bug in that the simple interface was not setting the tolerance… Fixed now.)